databricks/koalas/tests/test_series.py

```diff
@@ -155,6 +155,21 @@ def test_isin(self):
         with self.assertRaisesRegex(TypeError, msg):
             ds.isin(1)
 
+    def test_fillna(self):
+        ps = pd.Series([np.nan, 2, 3, 4, np.nan, 6], name='x')
+        ks = koalas.from_pandas(ps)
+
+        self.assert_eq(ks.fillna(0), ps.fillna(0))
+        self.assert_eq(ks.fillna({'x': 0}), ps.fillna(0))
```

Comment:
It turns out that `Pandas.Series.fillna` supports neither `Series` nor `dict` as value. 
`Koalas.Series.fillna` does!